backend/src/routes/routes.js

```diff
@@ -0,0 +1,37 @@
+const express = require("express")
+const router = express.Router()
+const path = require("path")
+
+const CharacterController = require("../controllers/character.api.js")
+const EpisodeController = require("../controllers/episode.api.js")
+const ApiController = require("../controllers/api.js")
+
+const {
+  getCharacterAvatar,
+  getAllCharacters,
+  getSingleCharacter
+} = CharacterController
+
+const {
+  getEpisodeAvatar,
+  getAllEpisodes,
+  getSingleEpisode
+} = EpisodeController
+
+const { getAPI } = ApiController
+
+router.get("/character/avatar", getCharacterAvatar)
```

Comment:
I tried to see what response returns this endpoint in master but I didn't get anything,
As you can see, I moved the characters and episodes images endpoint
https://github.com/lelouchB/final-space-api/blob/main/backend/server.js#L34-L37
```js
app.use(
  "/api/character/avatar",
  express.static(path.join(__dirname, "images/character")),
)
```
to the controller and I returns the path to the image instead of returning the image file (I think it is as it works currently)